cmd/context/utils.go

```diff
@@ -159,6 +159,11 @@ func LoadManifestWithContext(ctx context.Context, opts ManifestOptions) (*model.
 		return nil, err
 	}
 
+	// We need to read it again to propagate secrets env vars
```

Comment:
why is the manifest loaded before this point? to the `context` and `namespace` from the manifest if needed?

Reply:
Yes, we need it for the context resource which is just the context and the namespace